invenio_rdm_migrator/streams/communities/table_generator.py

```diff
@@ -22,7 +22,7 @@ def _generate_members_uuids(data):
 def _generate_featured_community_id(data):
     if not data.get("featured_community"):
         return None
-    return pid_pk(data)
```

Comment:
to be generic and consistent with others I think we should change the signature rather than the call, WDYT?